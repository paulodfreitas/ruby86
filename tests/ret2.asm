.module entry_code
.pseg
.org 0
    irmovl $100, %esp
    call Proc
    irmovl $10, %edx
    halt
Proc:
    ret
    rrmovl %edx, %ebx
.end
