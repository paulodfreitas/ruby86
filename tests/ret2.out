@ 0000
30
84
00
01
00
00
80
12
00
00
00
30
82
10
00
00
00
10
90
20
23
@ 0015
